reference/docs-conceptual/community/contributing/get-started-writing.md

```diff
@@ -0,0 +1,95 @@
+cmdlet reference only for the modules that ship with PowerShell. This content is also used to create
+the help information displayed by the `Get-Help` cmdlet.
+
+> [!NOTE]
+> The Table of Contents (TOC) for reference content is autogenerated by the publishing system. You
+> don't have to update the TOC.
+
+### Conceptual content
+
+The conceptual documentation includes the following content:
+
+- Release notes
+- Setup instructions
+- Sample scripts and how-to articles
+- DSC documentation
+- SDK documentation
+
+The [conceptual documentation][conceptual] is not organized by version. All articles are displayed
+for every version of PowerShell. We are working to create version-specific articles. When that
+feature is available in our documentation, this guide will be update with the appropriate
+information.
+
+> [!NOTE]
+> Anytime a conceptual article is added, removed, or renamed, the TOC must be updated and included
+> in the pull request.
+
+## Using git branches
+
+The default branch for PowerShell-Docs is the `staging` branch. Changes made in working branches are
+merged into the `staging` branch before the being published. About once a week, the `staging` branch
+is merged into the `live` branch. The `live` branch contains the content that is published to
+docs.microsoft.com. Changes should never be made directly in the `live` branch.
+
+If you are submitting a change to documentation that only applies to an unreleased version of
+PowerShell, check to see if there is a release branch for that version. All changes that apply to
+a specific, future version should be targeted at the release branch. Release branches have the
+following name pattern: `release-<version>`.
+
+Before starting any changes, create a working branch in your local copy of the PowerShell-Docs
+repository. This should be a [clone of your fork][fork]. Be sure to synchronize your local
+repository before creating your working branch. The working branch should be created from an
+update-to-date copy of the `staging` or `release` branch.
+
+Make the changes you want to submit following the process in the
+[Making your change][[making-changes] section of the central Contributor's Guide.
+
+### Creating new topics
+
+A GitHub issue must be created for any new document you want to contribute. Check for existing
+issues to make sure you're not duplicating efforts. Issues that are assigned to someone are
+considered to be "in progress". If you wish to collaborate on an issue, contact the person assigned
+to the issue.
+
+Similar to the PowerShell [RFC process][rfc], creating an issue before writing the content ensures
+that you don't spend a lot of time and effort on something that gets rejected by the PowerShell-Docs
+team. This also allows us to consult with you on the scope of the content and where is should fit
+within the PowerShell documentation.
+
+### Updating topics that existing articles
```

Comment:
Feels like a word is missing from this title.

Reply:
fixed